core/src/test/java/com/ibm/wala/core/tests/cha/JunitEntrypointSupportTest.java

```diff
@@ -0,0 +1,38 @@
+package com.ibm.wala.core.tests.cha;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.ibm.wala.core.tests.callGraph.CallGraphTestUtil;
+import com.ibm.wala.core.tests.util.TestConstants;
+import com.ibm.wala.core.util.scope.JUnitEntryPoints;
+import com.ibm.wala.ipa.callgraph.AnalysisScope;
+import com.ibm.wala.ipa.callgraph.Entrypoint;
+import com.ibm.wala.ipa.cha.ClassHierarchy;
+import com.ibm.wala.ipa.cha.ClassHierarchyException;
+import com.ibm.wala.ipa.cha.ClassHierarchyFactory;
+import com.ibm.wala.util.CancelException;
+import com.ibm.wala.util.collections.Iterator2Collection;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class JunitEntrypointSupportTest {
+
+  @Test
+  public void basic()
+      throws ClassHierarchyException, IllegalArgumentException, CancelException, IOException {
```

Comment:
I'm seeing a compilation failure reported on this line when using ECJ:

```
The declared exception CancelException is not actually thrown by the method basic() from type JunitEntrypointSupportTest
```

Reply:
Yup, my bad 🙂 Should be fixed now.